plugins/modules/aci_epg_to_contract.py

```diff
@@ -46,7 +46,8 @@
     - QoS class.
     - The APIC defaults to C(unspecified) when unset during creation.
     type: str
-    choices: [ level1, level2, level3, unspecified ]
+    choices: [ level1, level2, level3, level4, level5, level6, unspecified ]
+    default: unspecified
```

Comment:
Don't specify a default. It will be defaulted when not specified by the APIC itself. If you specify a default if the user does not specify it but the object already exist we might override the value.